### PR TITLE
[doc] Add CSCS webinar resources + links

### PR DESCRIPTION
**Pull request by vkarak**
This is still draft, until we get the link for the recording.

### CODE REVIEW

**Comment by victorusu** [comment]
The tutorial link is https://www.cscs.ch/publications/tutorials/2022/video-of-the-webinar-reframe-efficient-system-and-application-performance-testing/

**Review by teojgo — CHANGES_REQUESTED** [review]
The CI fails because of an obsolete pgi PrgEnv in the tutorial settings which are used to test the tutorial examples. Can you remove it?

File: tutorials/cscs-webinar-2022/config/mysettings.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+#
+# Generic fallback configuration
+#
+
+site_configuration = {
+    'systems': [
+        {
+            'name': 'tresa',
+            'descr': 'My laptopp',
```
Comment (teojgo) [inline tutorials/cscs-webinar-2022/config/mysettings.py:14]:
```suggestion
            'descr': 'My laptop',
```

**Comment by vkarak** [comment]
@teojgo I've changed all references to `pgi` in the tutorial (text, listings, code, figures).

**Review by teojgo — APPROVED** [review]

**Pull request merged**